docs/manual/src/docs/asciidoc/_includes/servlet/hello/index.adoc

```diff
@@ -5,7 +5,7 @@ This section covers the minimum setup for how to use Spring Security with Spring
 
 [NOTE]
 ====
-The completed application can be found at {gh-samples-url}/boot/helloworld[samples/boot/helloworld]
+The completed application can be found {gh-samples-url}/servlet/spring-boot/java/hello-security[here].
```

Comment:
Instead of "here", let's make the link more informative, like maybe "in our samples repository".